example.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<body>
+<h1>Publish Subscribe on Encrypted channel example</h1>
+<button id="publish">Publish encrypted messages</button>
+<br><br>
+<textarea id="subscriptions" style="width: 100%" rows="15"></textarea>
+</body>
+
+<!-- Include the latest Ably Library  -->
+<script src="https://cdn.ably.io/lib/ably.min.js"></script>
+
+<!-- Instance the Ably library  -->
+<script type="text/javascript">
+    var apiKey = 'INSERT-YOUR-API-KEY-HERE';
+    var channelName = 'encrypted:messages';
+
+    function createChannel(channelName, channelOpts) {
+        var realtime = new Ably.Realtime(apiKey);
+        return realtime.channels.get(channelName, channelOpts);
```

Comment:
I see why you've done it this way (so the publisher and subscriber get different Realtime instances so it's more of a real test), but still, I think this could well mislead someone into believing they should be creating a new Realtime instance for each channel they use (especially given the method name), which would be a big problem. I think the way you did it in the nodejs tutorial is better (just creating one Realtime instance at the top and using that).

Reply:
Makes sense. I was on the fence doing it this way too.
I'll update this.